### DISCUSSION TITLE
On the latest version of supabase CLI, how is the correct way to start my project locally?

### DISCUSSION THREAD

**Discussion by HenriqueRamos13** [supabase/supabase#13804] [Questions]
I was using an old version of supabase and my script to start was `npx supabase start`. This shows me a old version of the supabase studio for example, without logs section and icons left to table names. Other thing is when i ran this command the migrations is loaded automatically.

Now i upgraded my cli to latest version and i tried to run the command `supabase start`. Now the studio layout is different (close to the supabase on production) but the migrations dont run automatically (i need to run `supabase db reset` to execute all) and my storage looks broken. When i upload a file it didnt run my triggers to update profile table and when i go to the studio and click on the storage table, my file is on there but when i click on the file it does not load, still a gray background.

How is the correctly way to run the latest version of supabase with nextjs?

**Comment by Ziinc** [comment]
Hi Henrique, it would be helpful to provide a reproducable repo as well as the cli version(s) that you use so that we can try to replicate such bugs.
